umap/static/umap/js/modules/data/features.js

```diff
@@ -607,6 +607,14 @@ export class Point extends Feature {
     }
   }
 
+  _getLatLngs() {
+    return this.ui.getLatLng()
+  }
+
+  _setLatLngs(latlng) {
+    this.ui.setLatLng(latlng)
```

Comment:
That one doesn't return the result of the operation, not sure if it's intentional?